parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveStringifier.java

```diff
@@ -448,4 +449,16 @@ private void appendHex(byte[] array, int offset, int length, StringBuilder build
       }
     }
   };
+
+  static final PrimitiveStringifier FLOAT16_STRINGIFIER = new BinaryStringifierBase("FLOAT16_STRINGIFIER") {
+
+    @Override
+    String stringifyNotNull(Binary value) {
+      if (value.length() != 2) {
+        return BINARY_INVALID;
+      }
+      ByteBuffer buffer = value.toByteBuffer().order(ByteOrder.LITTLE_ENDIAN);
+      return DEFAULT_STRINGIFIER.stringify(toFloat(buffer.getShort(buffer.position())));
```

Comment:
Should we wrap this by adding a `Float16.toString()`?

Reply:
Added a Float16.toFloatString(..) method and please help check if it makes sense?